README.md

```diff
@@ -1,142 +1,136 @@
+```javascript
+export const hello = async (event, context) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify({
+      message: `Go Serverless v1.0! ${await message({
+        time: 1,
+        copy: "Your function executed successfully!"
+      })}`,
+      input: event
+    })
+  };
+};
 
-``` bash
-$ serverless offline start
+const message = ({ time, ...rest }) =>
+  new Promise((resolve, reject) =>
+    setTimeout(() => {
+      resolve(`${rest.copy} (with a delay)`);
+    }, time * 1000)
+  );
 ```
 
-Deploy your project
-
-``` bash
-$ serverless deploy
-```
+#### Environment Variables
 
-Deploy a single function
+To add environment variables to your project
 
-``` bash
-$ serverless deploy function --function hello
-```
+1. Rename `env.example` to `.env`.
+2. Add environment variables for your local stage to `.env`.
+3. Uncomment `environment:` block in the `serverless.yml` and reference the environment variable as `${env:MY_ENV_VAR}`. Where `MY_ENV_VAR` is added to your `.env` file.
+4. Make sure to not commit your `.env`.
 
 #### Running Tests
 
 Run your tests using
 
-``` bash
+```bash
 $ npm test
 ```
 
-We use Jest to run our tests. You can read more about setting up your tests [here](https://facebook.github.io/jest/docs/en/getting-started.html#content).
+We use Jest to run our tests. You can read more about setting up your tests [here](https://facebook.github.io/jest/docs/en/getting-started.html#content)
 
-#### Environment Variables
+#### Deployment
 
-To add environment variables to your project
+1. Deploy locally with:
 
-1. Rename `env.example` to `.env`.
-2. Add environment variables for your local stage to `.env`.
-3. Uncomment `environment:` block in the `serverless.yml` and reference the environment variable as `${env:MY_ENV_VAR}`. Where `MY_ENV_VAR` is added to your `.env` file.
-4. Make sure to not commit your `.env`.
-
-#### Linting
-
-We use [ESLint](https://eslint.org) to lint your code via the [serverless-bundle](https://github.com/AnomalyInnovations/serverless-bundle) plugin.
+```bash
+$ serverless deploy
+```
 
-You can turn this off by adding the following to your `serverless.yml`.
+2. Deploy only one function:
 
-``` yaml
-custom:
-  bundle:
-    linting: false
+```bash
+$ serverless deploy function --function hello
 ```
 
-To [override the default config](https://eslint.org/docs/user-guide/configuring), add a `.eslintrc.json` file. To ignore ESLint for specific files, add it to a `.eslintignore` file.
+3. Deploy with github actions
 
-### Support
+Please swap out your cloud provider details in **serverless.yml** and also in **./github/workflows/main.yml**.
 
-- Open a [new issue](https://github.com/AnomalyInnovations/serverless-nodejs-starter/issues/new) if you've found a bug or have some suggestions.
-- Or submit a pull request!
+For more information please visit: [info](https://github.com/features/actions)
 
----
+## Authors
```

Comment:
(you dont have to do this, i just wanted to make you aware of it)

checkout https://github.com/all-contributors/all-contributors

will allow you to acknowledge anyone who contributes, not just code.

Reply:
@all-contributors please add josephgarner for code